### ISSUE TITLE
logstash-5.1.1: cannot install logstash-core-event-java-5.1.1-java

### ISSUE THREAD

**Issue by domak** [elastic/logstash#6396]
- Version: 5.1.1
- Operating System: Red Hat Enterprise Linux Server release 7.2 (Maipo)
- Apache Maven 3.3.9 behind nexus proxy
- Java version: 1.8.0_51, vendor: Oracle Corporation
- Steps to Reproduce:

Hi,
I'm trying to install logstash-input-jmx and I get an error: 
```
Installing jmx4r 0.1.4
0:  jmx4r (0.1.4) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/jmx4r-0.1.4.gemspec
Using jrjackson 0.4.2
0:  jrjackson (0.4.2) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/jrjackson-0.4.2-java.gemspec
Using jrmonitor 0.4.2
0:  jrmonitor (0.4.2) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/jrmonitor-0.4.2.gemspec
Using jruby-openssl 0.9.16
0:  jruby-openssl (0.9.16) from /slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/lib/ruby/gems/shared/specifications/default/jruby-openssl-0.9.16-java.gemspec
Using jruby-stdin-channel 0.2.0
0:  jruby-stdin-channel (0.2.0) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/jruby-stdin-channel-0.2.0-java.gemspec
Using ruby-maven-libs 3.3.9
0:  ruby-maven-libs (3.3.9) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/ruby-maven-libs-3.3.9.gemspec
Using ruby-maven 3.3.12
0:  ruby-maven (3.3.12) from /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/specifications/ruby-maven-3.3.12.gemspec
  jar dependencies for logstash-core-event-java-5.1.1-java.gemspec . . .

NoMethodError: undefined method `includes?' for #<String:0x101baa6e>
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_settings.rb:110:in `setup_interpolated_settings'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_settings.rb:58:in `effective_settings'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_factory.rb:74:in `setup'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_factory.rb:51:in `maven_new'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_exec.rb:66:in `resolve_dependencies_list'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:237:in `install_dependencies'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:226:in `do_install'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:172:in `vendor_jars'
/slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/lib/ruby/shared/jars/post_install_hook.rb:28:in `(root)'
org/jruby/RubyProc.java:281:in `call'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:215:in `run_hooks'
org/jruby/RubyArray.java:1613:in `each'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:214:in `run_hooks'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:195:in `generate_bin'
org/jruby/RubyDir.java:341:in `chdir'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/shared_helpers.rb:53:in `chdir'
/slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/monitor.rb:211:in `mon_synchronize'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/shared_helpers.rb:52:in `chdir'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:189:in `generate_bin'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:73:in `install'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:107:in `install_gem_from_spec'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:269:in `install_sequentially'
org/jruby/RubyArray.java:1613:in `each'
/slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/forwardable.rb:201:in `each'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:268:in `install_sequentially'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:90:in `run'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:18:in `install'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli/install.rb:80:in `run'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:155:in `install'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:10:in `start'
/slbafrw55tn01/appli/elk/logstash-5.1.1/lib/bootstrap/bundler.rb:152:in `execute_bundler'
/slbafrw55tn01/appli/elk/logstash-5.1.1/lib/bootstrap/bundler.rb:113:in `invoke!'
/slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/install.rb:139:in `install_gems_list!'
/slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/install.rb:58:in `execute'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11:in `execute'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:132:in `run'
/slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/main.rb:46:in `(root)'
Bundler::InstallError: An error occurred while installing logstash-core-event-java (5.1.1), and Bundler cannot continue.
Make sure that `gem install logstash-core-event-java -v '5.1.1'` succeeds before bundling.
  install_gem_from_spec at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:145
   install_sequentially at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:269
                   each at org/jruby/RubyArray.java:1613
                   each at /slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/forwardable.rb:201
   install_sequentially at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:268
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:90
                install at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:18
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli/install.rb:80
                install at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:155
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/command.rb:27
         invoke_command at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:126
               dispatch at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor.rb:359
                  start at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/base.rb:440
                  start at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:10
        execute_bundler at /slbafrw55tn01/appli/elk/logstash-5.1.1/lib/bootstrap/bundler.rb:152
                invoke! at /slbafrw55tn01/appli/elk/logstash-5.1.1/lib/bootstrap/bundler.rb:113
     install_gems_list! at /slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/install.rb:139
                execute at /slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/install.rb:58
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                execute at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                    run at /slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:132
                 (root) at /slbafrw55tn01/appli/elk/logstash/lib/pluginmanager/main.rb:46
```

So I've tried to install logstash-core-event-java as mentioned in the log:

```
$ export PATH=/slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/bin/:$PATH

$ /slbafrw55tn01/appli/elk/logstash-5.1.1/vendor/jruby/bin/gem install logstash-core-event-java -v '5.1.1'
  jar dependencies for logstash-core-event-java-5.1.1-java.gemspec . . .
      com.fasterxml.jackson.core:jackson-core:2.7.3
      com.fasterxml.jackson.core:jackson-databind:2.7.3
      com.fasterxml.jackson.module:jackson-module-afterburner:2.7.3
      com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.7.3
      org.apache.logging.log4j:log4j-api:2.6.2
Successfully installed logstash-core-event-java-5.1.1-java
1 gem installed
```

I'm behind a proxy (http_proxy/https_proxy/HTTP_PROXY/HTTPS_PROXY all set) and maven use a nexus proxy.
Is there some thing on my side?

**Comment by domak** [comment]
I've tried on windows and get an error on the same part (but with a different message):
```
Using ruby-maven-libs 3.3.9
0:  ruby-maven-libs (3.3.9) from c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/specifications/ruby-maven-libs-3.3.9.gemspec
Using ruby-maven 3.3.12
0:  ruby-maven (3.3.12) from c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/specifications/ruby-maven-3.3.12.gemspec
  jar dependencies for logstash-core-event-java-5.1.1-java.gemspec . . .
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-parseable POM c:\work\temp\risk\logstash-5.1.1\vendor\bundle\jruby\1.9\gems\jar-dependencies-0.3.7\lib\jars\gemspec_pom.rb: only whitespace content allowed before s
t tag and not # (position: START_DOCUMENT seen #... @1:1)  @ line 1, column 1
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project  (c:\work\temp\risk\logstash-5.1.1\vendor\bundle\jruby\1.9\gems\jar-dependencies-0.3.7\lib\jars\gemspec_pom.rb) has 1 error
[ERROR]     Non-parseable POM c:\work\temp\risk\logstash-5.1.1\vendor\bundle\jruby\1.9\gems\jar-dependencies-0.3.7\lib\jars\gemspec_pom.rb: only whitespace content allowed befo
start tag and not # (position: START_DOCUMENT seen #... @1:1)  @ line 1, column 1 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/ModelParseException

Errno::ENOENT: No such file or directory - C:/work/temp/risk/logstash-5.1.1/logstash-core-event-java/deps.lst
org/jruby/RubyIO.java:3804:in `read'
org/jruby/RubyIO.java:3987:in `read'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:73:in `load_from_maven'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:239:in `install_dependencies'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:226:in `do_install'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/installer.rb:172:in `vendor_jars'
C:/work/temp/risk/logstash-5.1.1/vendor/jruby/lib/ruby/shared/jars/post_install_hook.rb:28:in `(root)'
org/jruby/RubyProc.java:281:in `call'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:215:in `run_hooks'
org/jruby/RubyArray.java:1613:in `each'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:214:in `run_hooks'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:195:in `generate_bin'
org/jruby/RubyDir.java:341:in `chdir'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/shared_helpers.rb:53:in `chdir'
C:/work/temp/risk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/monitor.rb:211:in `mon_synchronize'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/shared_helpers.rb:52:in `chdir'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:189:in `generate_bin'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/source/path.rb:73:in `install'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:107:in `install_gem_from_spec'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:269:in `install_sequentially'
org/jruby/RubyArray.java:1613:in `each'
C:/work/temp/risk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/forwardable.rb:201:in `each'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:268:in `install_sequentially'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:90:in `run'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:18:in `install'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli/install.rb:80:in `run'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:155:in `install'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:10:in `start'
C:/work/temp/risk/logstash-5.1.1/lib/bootstrap/bundler.rb:152:in `execute_bundler'
C:/work/temp/risk/logstash-5.1.1/lib/bootstrap/bundler.rb:113:in `invoke!'
c:/work/temp/risk/logstash-5.1.1/lib/pluginmanager/install.rb:139:in `install_gems_list!'
c:/work/temp/risk/logstash-5.1.1/lib/pluginmanager/install.rb:58:in `execute'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11:in `execute'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67:in `run'
c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:132:in `run'
c:\work\temp\risk\logstash-5.1.1\lib\pluginmanager\main.rb:46:in `(root)'
Bundler::InstallError: An error occurred while installing logstash-core-event-java (5.1.1), and Bundler cannot continue.
Make sure that `gem install logstash-core-event-java -v '5.1.1'` succeeds before bundling.
  install_gem_from_spec at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:145
  install_gem_from_spec at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:126
   install_sequentially at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:269
                   each at org/jruby/RubyArray.java:1613
                   each at C:/work/temp/risk/logstash-5.1.1/vendor/jruby/lib/ruby/1.9/forwardable.rb:201
   install_sequentially at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:268
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:90
                install at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/installer.rb:18
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli/install.rb:80
                install at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:155
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/command.rb:27
         invoke_command at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:126
               dispatch at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor.rb:359
                  start at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/vendor/thor/lib/thor/base.rb:440
                  start at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/cli.rb:10
        execute_bundler at C:/work/temp/risk/logstash-5.1.1/lib/bootstrap/bundler.rb:152
                invoke! at C:/work/temp/risk/logstash-5.1.1/lib/bootstrap/bundler.rb:113
     install_gems_list! at c:/work/temp/risk/logstash-5.1.1/lib/pluginmanager/install.rb:139
                execute at c:/work/temp/risk/logstash-5.1.1/lib/pluginmanager/install.rb:58
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                execute at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/subcommand/execution.rb:11
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:67
                    run at c:/work/temp/risk/logstash-5.1.1/vendor/bundle/jruby/1.9/gems/clamp-0.6.5/lib/clamp/command.rb:132
                 (root) at c:\work\temp\risk\logstash-5.1.1\lib\pluginmanager\main.rb:46
```

**Comment by suyograo** [comment]
This looks related to https://github.com/mkristian/jar-dependencies/issues/16 which was published in jar-dependencies 0.3.7.

@mkristian can you please take a look?

NoMethodError: undefined method `includes?' for #<String:0x101baa6e>
/slbafrw55tn01/appli/elk/logstash/vendor/bundle/jruby/1.9/gems/jar-dependencies-0.3.7/lib/jars/maven_settings.rb:110:in `setup_interpolated_settings'

**Comment by mkristian** [comment]
pushes a fixed jar-depedencies-0.3.8 to rubgems.org

**Comment by domak** [comment]
It works! Thanks a lot!
